sdk/storage/storage-queue/src/utils/tracing.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getTracer, Span, SpanOptions, SpanKind } from "@azure/core-tracing";
+
+/**
+ * Creates a span using the global tracer.
+ * @param name The name of the operation being performed.
+ * @param options The options for the underlying http request.
+ */
+export function createSpan(
+  operationName: string,
+  options: SpanOptions = {}
+): { span: Span; spanOptions: SpanOptions } {
+  const tracer = getTracer();
+  const spanOptions: SpanOptions = {
+    ...options,
+    kind: SpanKind.CLIENT
+  };
+
+  const span = tracer.startSpan(`Azure.Storage.Queue.${operationName}`, spanOptions);
```

Comment:
>Azure.Storage.Queu [](start = 33, length = 18)

nit: would it be better to have some part of `createSpan` in core-tracing? I see we still like some wrapper here for convenience but there's also common code that is not storage-specific.

Reply:
👍 Yes, I'm going to look at how we can re-use more of this in the future and I'll clean it up in the consuming packages when I do!